db/seeds.rb

```diff
@@ -7,3 +7,141 @@
 #   ["Action", "Comedy", "Drama", "Horror"].each do |genre_name|
 #     MovieGenre.find_or_create_by!(name: genre_name)
 #   end
+
+require 'faker'
+
+# API or website (scrape) with street address, postcode and a deescription
+
+puts "Cleaning user database"
+User.destroy_all
+
+puts "Seeding user DB"
+test_user = User.new(
+  email: "test@test.com",
+  password: "123456",
+  username: "magnus-jr"
+)
+test_user.save
+
+
+User.all.each do |user|
+  user.image.attach(
+    io: URI.open(profile_photos.sample),
+    filename: "profile.jpg",
+    content_type: "image/jpeg"
+  )
+  user.save!
+end
+
+profile_photos = [
+  "https://avatars.githubusercontent.com/u/73750355?v=4",
+  "https://d26jy9fbi4q9wx.cloudfront.net/rails/active_storage/representations/proxy/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBeXB2QXc9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--ec570f8e7ba217d7dd50b05f8c190fd677268763/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaDdCem9MWm05eWJXRjBTU0lJYW5CbkJqb0dSVlE2RTNKbGMybDZaVjkwYjE5bWFXeHNXd2hwQWNocEFjaDdCam9KWTNKdmNEb09ZWFIwWlc1MGFXOXUiLCJleHAiOm51bGwsInB1ciI6InZhcmlhdGlvbiJ9fQ==--b67d9ded4d28d0969fbb98b4c21b79257705a99a/IMG_9339.jpg",
+  "https://avatars.githubusercontent.com/u/183761743?v=4",
+]
+
+puts "Users seeded!"
+
+Property.destroy_all
+puts "Cleaning properties DB"
+puts "Seeding properties DB"
+
+
+CSV.foreach("lib/W_addresses.csv", headers: true) do |address|
+  types = ["flat","house", "terraced house", "semi-detached house"]
+  address_hash = {
+    street_address:  address["Address"],
+    postcode: address["Postcode"],
+    description: Faker::Lorem.paragraph(1),
+    bedrooms: rand(1..4),
+    bathrooms: rand(1..3),
+    garden: ["true", "false"].sample,
+    image_urls: Faker::LoremFlickr.image,
+    council_tax: ["A", "B", "C", "D", "E"].sample,
+    type: types.sample,
+    floor_area: rand(45.0...120.0)
+  }
+  Property.create!(address_hash)
+end
+
```

Comment:
Just add a line confirming that the models are properly generated.

I.e. `puts Property.all.first`